### PR TITLE
Cat Person Roundstart Trait

### PR DESCRIPTION
**Pull request by Kapu1178**
## About the PR 
The glorious underused catperson of gooncode is now available in game.

## Why's this needed? 
NYAAAAAAAA

## Changelog

```
(u)Kapu1178
(*)Catperson is added to the traits menu for -1 point.
```

### CODE REVIEW

**Comment by yyzsong** [comment]
agony

**Comment by comradef191** [comment]
oh no.
dont do it kapu
have mercy on them, they need not witness our catperson. i beg of you.

**Comment by Gannets** [comment]
I drew these sprites as an April fools joke back in 2017.

**Comment by DatBoiTim** [comment]
Dear god. First Open Source PR is Cat Folks.

**Comment by TTerc** [comment]
just make it -8

**Comment by PowerfulBacon** [comment]
It was only a matter of time.

File: code/datums/traits.dm [hunk 1]
```diff
@@ -1118,3 +1118,20 @@ obj/trait/pilot
 			var/mob/living/carbon/human/H = owner
 			H.set_mutantrace(/datum/mutantrace/roach)
 		return
+
+/obj/trait/cat
+	name = "Feline (-1) \[Species\]"
+	cleanName = "Cat"
+	icon_state = "placeholder"
```
Comment (PowerfulBacon) [inline code/datums/traits.dm:1125]: placeholder?

Comment (Kapu1178) [reply code/datums/traits.dm:1125]: Placeholder is a "?" in the same style as the other trait icons. I thought it was more fitting than reusing cateye or making a new icon and failing to replicate the style, and that it complimented the joke.

**Comment by francinum** [comment]
🔑

**Comment by MCterra10** [comment]
:key:

**Comment by DatBoiTim** [comment]
NO FELINIDS OF CLOVER! REJOICE!!

**Pull request closed without merging**